sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_client.hpp

```diff
@@ -16,8 +16,8 @@
 #include "azure/storage/blobs/protocol/blob_rest_client.hpp"
 
 namespace Azure { namespace Storage { namespace Files { namespace DataLake {
-  class DirectoryClient;
-  class FileClient;
+  class DataLakeDirectoryClient;
```

Comment:
There's no benefit then to have the `DataLake` namespace right?  

IMHO, its simpler and clearer whithout the prefix.  The prefix makes it look like pure C.

Reply:
Yes, as Rick mentioned this morning, it is purely for searching promotion.